apis/v1beta1/condition_consts.go

```diff
@@ -83,6 +83,11 @@ const (
 	// are automatically re-tried by the controller.
 	PoweringOnFailedReason = "PoweringOnFailed"
 
+	// NotFoundByBIOSUUIDReason (Severity=Error) documents a VSphereVM which can't be found by BIOS UUID.
```

Comment:
```suggestion
	// NotFoundByBIOSUUIDReason (Severity=Warning) documents a VSphereVM which can't be found by BIOS UUID.
```

Let's use warning here. 

> A recoverable error occurred during provisioning and someone/something should take action (warning)
> A catastrophic error occurred (error)
> https://github.com/kubernetes-sigs/cluster-api/blob/9377ba2a19ea69f716f88689f6a550698d4da4b2/docs/proposals/20200506-conditions.md#the-clusterinfrastructureready-condition

Would be also same severity as PoweringOnFailedReason

Reply:
agree, will update in next patch